scrapy/exporters.py

```diff
@@ -84,19 +85,22 @@ class JsonLinesItemExporter(BaseItemExporter):
     def __init__(self, file, **kwargs):
         self._configure(kwargs, dont_fail=True)
         self.file = file
-        self.encoder = ScrapyJSONEncoder(**kwargs)
+        ensure_ascii = not self.encoding
```

Comment:
It would be better to do this as:
```
kwargs.setdefault('ensure_ascii', not self.encoding)`
```
and removing the explicit `ensure_ascii` from the ScrapyJSONEncoder call.

This way it won't break subclasses that are already be passing `ensure_ascii` for this in kwargs.